src/GoProControl.cpp

```diff
@@ -1649,7 +1693,8 @@ uint8_t GoProControl::connectClient()
   }
 }
 
-bool GoProControl::listenResponse()
+
+bool GoProControl::listenResponse(bool mediatimer)
```

Comment:
make it const here too and remove the newline

Reply:
ok